#' RDP training set v19
#'
#' The sequence and taxonomy data for the 24,642 sequences found in the
#' Ribosomal Database Project's trainset19_072023 training set for use with the
#' naive Bayesian classifier as implemented in the `{phylyotypr}` R package.
#' Originally released by the RDP in July 2023. The `rdp` version contains the
#' same sequences as provided by the official RDP version (24,642 total
#' sequences). The `pds` version contains extra eukaryotic sequences including
#' chloroplasts and mitochondria (24,762 total sequences). See the mothur
#' reference file page in "Sources" for more information.
#'
#' @format
#' A data frame with 3 columns. Each row represents a different
#' sequence:
#' \describe{
#'   \item{id}{Sequence accession identifier}
#'   \item{sequence}{DNA sequence string}
#'   \item{taxonomy}{Taxonomic string with each level separated with a `;`}
#' }
#'
#' @source
#' * [mothur-formatted files](https://mothur.org/wiki/rdp_reference_files/)
#' * [Description of how mothur-formatted files were generated](https://mothur.org/blog/2024/RDP-v19-reference-files/)
#' * [RDP sourceforge page](https://sourceforge.net/p/rdp-classifier/news/2023/08/rdp-classifier-214-august-2023-released/)
#'
"rdp_trainset19"


#' @rdname rdp_trainset19
"rdp_trainset19_pds"



#' RDP training set v18
#'
#' The sequence and taxonomy data for the 21,313 sequences found in the
#' Ribosomal Database Project's trainset18_062020 training set for use with the
#' naive Bayesian classifier as implemented in the `{phylyotypr}` R package.
#' Originally released by the RDP in June 2020. The `rdp` version contains the
#' same sequences as provided by the official RDP version. The `pds` version
#' contains 119 eukaryotic sequences including chloroplasts and mitochondria.
#' See the mothur reference file page in "Sources" for more information.
#'
#' @format
#' A data frame with 3 columns. Each row represents a different
#' sequence:
#' \describe{
#'   \item{id}{Sequence accession identifier}
#'   \item{sequence}{DNA sequence string}
#'   \item{taxonomy}{Taxonomic string with each level separated with a `;`}
#' }
#'
#' @source
#' * [mothur-formatted files](https://mothur.org/wiki/rdp_reference_files/)
#' * [Description of how mothur-formatted files were generated](https://mothur.org/blog/2021/RDP-v18-reference-files/)
#' * [RDP sourceforge page](https://sourceforge.net/projects/rdp-classifier/files/RDP_Classifier_TrainingData/)
#'
"rdp_trainset18"


#' @rdname rdp_trainset18
"rdp_trainset18_pds"



#' RDP training set v16
#'
#' The sequence and taxonomy data for the 13,212 sequences found in the
#' Ribosomal Database Project's trainset16_022016 training set for use with the
#' naive Bayesian classifier as implemented in the `{phylyotypr}` R package.
#' Originally released by the RDP in February 2016. The `rdp` version contains
# the same sequences as provided by the official RDP version. The `pds` version
# contains 119 eukaryotic sequences including chloroplasts and mitochondria. See
# the mothur reference file page in "Sources" for more information.
#'
#' @format
#' A data frame with 3 columns. Each row represents a different
#' sequence:
#' \describe{
#'   \item{id}{Sequence accession identifier}
#'   \item{sequence}{DNA sequence string}
#'   \item{taxonomy}{Taxonomic string with each level separated with a `;`}
#' }
#'
#' @source
#' * [mothur-formatted files](https://mothur.org/wiki/rdp_reference_files/)
#' * [Description of how mothur-formatted files were generated](https://mothur.org/blog/2017/RDP-v16-reference_files/)
#' * [RDP sourceforge page](https://sourceforge.net/projects/rdp-classifier/files/RDP_Classifier_TrainingData/)
#'
"rdp_trainset16"


#' @rdname rdp_trainset16
"rdp_trainset16_pds"




#' RDP training set v14
#'
#' The sequence and taxonomy data for the 10,679 sequences found in the
#' Ribosomal Database Project's trainset14_032015 training set for use with the
#' naive Bayesian classifier as implemented in the `{phylyotypr}` R package.
#' Originally released by the RDP in May 2015 The `rdp` version contains
# the same sequences as provided by the official RDP version. The `pds` version
# contains 119 eukaryotic sequences including chloroplasts and mitochondria. See
# the mothur reference file page in "Sources" for more information.
#'
#' @format
#' A data frame with 3 columns. Each row represents a different
#' sequence:
#' \describe{
#'   \item{id}{Sequence accession identifier}
#'   \item{sequence}{DNA sequence string}
#'   \item{taxonomy}{Taxonomic string with each level separated with a `;`}
#' }
#'
#' @source
#' * [mothur-formatted files](https://mothur.org/wiki/rdp_reference_files/)
#' * [Description of how mothur-formatted files were generated](https://mothur.org/blog/2015/RDP-v14-reference_files/)
#' * [RDP sourceforge page](https://sourceforge.net/projects/rdp-classifier/files/RDP_Classifier_TrainingData/)
#'
"rdp_trainset14"


#' @rdname rdp_trainset14
"rdp_trainset14_pds"




#' RDP training set v10
#'
#' The sequence and taxonomy data for the 10,650 sequences found in the
#' Ribosomal Database Project's trainset10_082014 training set for use with the
#' naive Bayesian classifier as implemented in the `{phylyotypr}` R package.
#' Originally released by the RDP in September 2014 The `rdp` version contains
# the same sequences as provided by the official RDP version. The `pds` version
# contains 119 eukaryotic sequences including chloroplasts and mitochondria. See
# the mothur reference file page in "Sources" for more information.
#'
#' @format
#' A data frame with 3 columns. Each row represents a different
#' sequence:
#' \describe{
#'   \item{id}{Sequence accession identifier}
#'   \item{sequence}{DNA sequence string}
#'   \item{taxonomy}{Taxonomic string with each level separated with a `;`}
#' }
#'
#' @source
#' * [mothur-formatted files](https://mothur.org/wiki/rdp_reference_files/)
#' * [Description of how mothur-formatted files were generated](https://mothur.org/blog/2014/RDP-v10-reference-files/)
#' * [RDP sourceforge page](https://sourceforge.net/projects/rdp-classifier/files/RDP_Classifier_TrainingData/)
#'
"rdp_trainset10"


#' @rdname rdp_trainset10
"rdp_trainset10_pds"



#' RDP training set v9
#'
#' The sequence and taxonomy data for the 10,049 sequences found in the
#' Ribosomal Database Project's trainset9_032012 training set for use with the
#' naive Bayesian classifier as implemented in the `{phylyotypr}` R package.
#' Originally released by the RDP in March 2012 The `rdp` version contains
# the same sequences as provided by the official RDP version. The `pds` version
# contains 119 eukaryotic sequences including chloroplasts and mitochondria.
#'
#' @format
#' A data frame with 3 columns. Each row represents a different
#' sequence:
#' \describe{
#'   \item{id}{Sequence accession identifier}
#'   \item{sequence}{DNA sequence string}
#'   \item{taxonomy}{Taxonomic string with each level separated with a `;`}
#' }
#'
#' @source
#' * [mothur-formatted files](https://mothur.org/wiki/rdp_reference_files/)
#' * [RDP sourceforge page](https://sourceforge.net/projects/rdp-classifier/files/RDP_Classifier_TrainingData/)
#'
"rdp_trainset9"


#' @rdname rdp_trainset9
"rdp_trainset9_pds"



#' RDP training set v7
#'
#' The sequence and taxonomy data for the 10,046 sequences found in the
#' Ribosomal Database Project's trainset7_112011 training set for use with the
#' naive Bayesian classifier as implemented in the `{phylyotypr}` R package.
#' Originally released by the RDP in November 2011 The `rdp` version contains
# the same sequences as provided by the official RDP version. The `pds` version
# contains 119 eukaryotic sequences including chloroplasts and mitochondria.
#'
#' @format
#' A data frame with 3 columns. Each row represents a different
#' sequence:
#' \describe{
#'   \item{id}{Sequence accession identifier}
#'   \item{sequence}{DNA sequence string}
#'   \item{taxonomy}{Taxonomic string with each level separated with a `;`}
#' }
#'
#' @source
#' * [mothur-formatted files](https://mothur.org/wiki/rdp_reference_files/)
#' * [RDP sourceforge page](https://sourceforge.net/projects/rdp-classifier/files/RDP_Classifier_TrainingData/)
#'
"rdp_trainset7"


#' @rdname rdp_trainset7
"rdp_trainset7_pds"



#' RDP training set v6
#'
#' The sequence and taxonomy data for the 10,046 sequences found in the
#' Ribosomal Database Project's trainset6_032010 training set for use with the
#' naive Bayesian classifier as implemented in the `{phylyotypr}` R package.
#' Originally released by the RDP in November 2011 The `rdp` version contains
# the same sequences as provided by the official RDP version. The `pds` version
# contains 119 eukaryotic sequences including chloroplasts and mitochondria.
#'
#' @format
#' A data frame with 3 columns. Each row represents a different
#' sequence:
#' \describe{
#'   \item{id}{Sequence accession identifier}
#'   \item{sequence}{DNA sequence string}
#'   \item{taxonomy}{Taxonomic string with each level separated with a `;`}
#' }
#'
#' @source
#' * [mothur-formatted files](https://mothur.org/wiki/rdp_reference_files/)
#' * [RDP sourceforge page](https://sourceforge.net/projects/rdp-classifier/files/RDP_Classifier_TrainingData/)
#'
"rdp_trainset6"


#' @rdname rdp_trainset6
"rdp_trainset6_pds"


#' Silva training set v138.2
#'
#' The sequence and taxonomy data for the full-length sequences found in Silva's
#' SSU (16S/18S) database version 138.2 for use with the naive Bayesian
#' classifier as implemented in the `{phylyotypr}` R package. This includes
#' sequences from bacteria, archaea, and eukaryotes. Originally released by
#' Silva in July 2024. The `nr` version contains the non-redundant version of
#' the database, which excludes sequences more than 99% similar to other
#' sequences (164,296 total sequences); some additional polishing of the
#' sequences and taxonomy have been applied. The `seed` version contains 8,707
#' sequences that are thought to be part of their SEED database for purposes of
#' aligning. The sequences provided here are not aligned. See the mothur
#' reference file page in "Sources" for more information.
#'
#' @format
#' A data frame with 3 columns. Each row represents a different
#' sequence:
#' \describe{
#'   \item{id}{Sequence accession identifier}
#'   \item{sequence}{DNA sequence string}
#'   \item{taxonomy}{Taxonomic string with each level separated with a `;`}
#' }
#'
#' @source
#' * [mothur-formatted files](https://mothur.org/wiki/silva_reference_files/)
#' * [Description of how mothur-formatted files were generated](https://mothur.org/blog/2024/SILVA-v138_2-reference-files/)
#' * [Silva release notes for this version](https://www.arb-silva.de/documentation/release-1382/)
#'
"silva_v138_2_nr"

#' @rdname silva_v138_2_nr
"silva_v138_2_seed"



#' Silva training set v138.1
#'
#' The sequence and taxonomy data for the full-length sequences found in Silva's
#' SSU (16S/18S) database version 138.1 for use with the naive Bayesian
#' classifier as implemented in the `{phylyotypr}` R package. This includes
#' sequences from bacteria, archaea, and eukaryotes. Originally released by
#' Silva in August 2020. The `nr` version contains the non-redundant version of
#' the database, which excludes sequences more than 99% similar to other
#' sequences (146,601 total sequences); some additional polishing of the
#' sequences and taxonomy have been applied. The `seed` version contains 7,641
#' sequences that are thought to be part of their SEED database for purposes of
#' aligning. The sequences provided here are not aligned. See the mothur
#' reference file page in "Sources" for more information.
#'
#' @format
#' A data frame with 3 columns. Each row represents a different
#' sequence:
#' \describe{
#'   \item{id}{Sequence accession identifier}
#'   \item{sequence}{DNA sequence string}
#'   \item{taxonomy}{Taxonomic string with each level separated with a `;`}
#' }
#'
#' @source
#' * [mothur-formatted files](https://mothur.org/wiki/silva_reference_files/)
#' * [Description of how mothur-formatted files were generated](https://mothur.org/blog/2021/SILVA-v138_1-reference-files/)
#' * [Silva release notes for this version](https://www.arb-silva.de/documentation/release-1381/)
"silva_v138_1_nr"

#' @rdname silva_v138_1_nr
"silva_v138_1_seed"



#' Silva training set v138
#'
#' The sequence and taxonomy data for the full-length sequences found in Silva's
#' SSU (16S/18S) database version 138 for use with the naive Bayesian
#' classifier as implemented in the `{phylyotypr}` R package. This includes
#' sequences from bacteria, archaea, and eukaryotes. Originally released by
#' Silva in December 2019. The `nr` version contains the non-redundant version of
#' the database, which excludes sequences more than 99% similar to other
#' sequences (146,796 total sequences); some additional polishing of the
#' sequences and taxonomy have been applied. The `seed` version contains 7,656
#' sequences that are thought to be part of their SEED database for purposes of
#' aligning. The sequences provided here are not aligned. See the mothur
#' reference file page in "Sources" for more information.
#'
#' @format
#' A data frame with 3 columns. Each row represents a different
#' sequence:
#' \describe{
#'   \item{id}{Sequence accession identifier}
#'   \item{sequence}{DNA sequence string}
#'   \item{taxonomy}{Taxonomic string with each level separated with a `;`}
#' }
#'
#' @source
#' * [mothur-formatted files](https://mothur.org/wiki/silva_reference_files/)
#' * [Description of how mothur-formatted files were generated](https://mothur.org/blog/2020/SILVA-v138-reference-files/)
#' * [Silva release notes for this version](https://www.arb-silva.de/documentation/release-138/)
"silva_v138_nr"

#' @rdname silva_v138_nr
"silva_v138_seed"


#' Silva training set v132
#'
#' The sequence and taxonomy data for the full-length sequences found in Silva's
#' SSU (16S/18S) database version 132 for use with the naive Bayesian
#' classifier as implemented in the `{phylyotypr}` R package. This includes
#' sequences from bacteria, archaea, and eukaryotes. Originally released by
#' Silva in December 2017. The `nr` version contains the non-redundant version of
#' the database, which excludes sequences more than 99% similar to other
#' sequences (213,119 total sequences); some additional polishing of the
#' sequences and taxonomy have been applied. The `seed` version contains 11,180
#' sequences that are thought to be part of their SEED database for purposes of
#' aligning. The sequences provided here are not aligned. See the mothur
#' reference file page in "Sources" for more information.
#'
#' @format
#' A data frame with 3 columns. Each row represents a different
#' sequence:
#' \describe{
#'   \item{id}{Sequence accession identifier}
#'   \item{sequence}{DNA sequence string}
#'   \item{taxonomy}{Taxonomic string with each level separated with a `;`}
#' }
#'
#' @source
#' * [mothur-formatted files](https://mothur.org/wiki/silva_reference_files/)
#' * [Description of how mothur-formatted files were generated](https://mothur.org/blog/2018/SILVA-v132-reference-files/)
#' * [Silva release notes for this version](https://www.arb-silva.de/documentation/release-132/)
"silva_v132_nr"

#' @rdname silva_v132_nr
"silva_v132_seed"


#' Silva training set v128
#'
#' The sequence and taxonomy data for the full-length sequences found in Silva's
#' SSU (16S/18S) database version 128 for use with the naive Bayesian
#' classifier as implemented in the `{phylyotypr}` R package. This includes
#' sequences from bacteria, archaea, and eukaryotes. Originally released by
#' Silva in September 2016. The `nr` version contains the non-redundant version of
#' the database, which excludes sequences more than 99% similar to other
#' sequences (190,661 total sequences); some additional polishing of the
#' sequences and taxonomy have been applied. The `seed` version contains 11,213
#' sequences that are thought to be part of their SEED database for purposes of
#' aligning. The sequences provided here are not aligned. See the mothur
#' reference file page in "Sources" for more information.
#'
#' @format
#' A data frame with 3 columns. Each row represents a different
#' sequence:
#' \describe{
#'   \item{id}{Sequence accession identifier}
#'   \item{sequence}{DNA sequence string}
#'   \item{taxonomy}{Taxonomic string with each level separated with a `;`}
#' }
#'
#' @source
#' * [mothur-formatted files](https://mothur.org/wiki/silva_reference_files/)
#' * [Description of how mothur-formatted files were generated](https://mothur.org/blog/2017/SILVA-v128-reference-files/)
#' * [Silva release notes for this version](https://www.arb-silva.de/documentation/release-128/)
"silva_v128_nr"

#' @rdname silva_v128_nr
"silva_v128_seed"




#' Silva training set v123
#'
#' The sequence and taxonomy data for the full-length sequences found in Silva's
#' SSU (16S/18S) database version 123 for use with the naive Bayesian
#' classifier as implemented in the `{phylyotypr}` R package. This includes
#' sequences from bacteria, archaea, and eukaryotes. Originally released by
#' Silva in July 2015. The `nr` version contains the non-redundant version of
#' the database, which excludes sequences more than 99% similar to other
#' sequences (172,418 total sequences); some additional polishing of the
#' sequences and taxonomy have been applied. The `seed` version contains 14,914
#' sequences that are thought to be part of their SEED database for purposes of
#' aligning. The sequences provided here are not aligned. See the mothur
#' reference file page in "Sources" for more information.
#'
#' @format
#' A data frame with 3 columns. Each row represents a different
#' sequence:
#' \describe{
#'   \item{id}{Sequence accession identifier}
#'   \item{sequence}{DNA sequence string}
#'   \item{taxonomy}{Taxonomic string with each level separated with a `;`}
#' }
#'
#' @source
#' * [mothur-formatted files](https://mothur.org/wiki/silva_reference_files/)
#' * [Description of how mothur-formatted files were generated](https://mothur.org/blog/2015/SILVA-v123-reference-files/)
#' * [Silva release notes for this version](https://www.arb-silva.de/documentation/release-123/)
"silva_v123_nr"

#' @rdname silva_v123_nr
"silva_v123_seed"



#' Silva training set v119
#'
#' The sequence and taxonomy data for the full-length sequences found in Silva's
#' SSU (16S/18S) database version 119 for use with the naive Bayesian
#' classifier as implemented in the `{phylyotypr}` R package. This includes
#' sequences from bacteria, archaea, and eukaryotes. Originally released by
#' Silva in July 2014. The `nr` version contains the non-redundant version of
#' the database, which excludes sequences more than 99% similar to other
#' sequences (153,307 total sequences); some additional polishing of the
#' sequences and taxonomy have been applied. The `seed` version contains 15,009
#' sequences that are thought to be part of their SEED database for purposes of
#' aligning. The sequences provided here are not aligned. See the mothur
#' reference file page in "Sources" for more information.
#'
#' @format
#' A data frame with 3 columns. Each row represents a different
#' sequence:
#' \describe{
#'   \item{id}{Sequence accession identifier}
#'   \item{sequence}{DNA sequence string}
#'   \item{taxonomy}{Taxonomic string with each level separated with a `;`}
#' }
#'
#' @source
#' * [mothur-formatted files](https://mothur.org/wiki/silva_reference_files/)
#' * [Description of how mothur-formatted files were generated](https://mothur.org/blog/2014/SILVA-v119-reference-files/)
#' * [Silva release notes for this version](https://www.arb-silva.de/documentation/release-119/)
"silva_v119_nr"

#' @rdname silva_v119_nr
"silva_v119_seed"




#' Greengenes training set 20_10
#'
#' The sequence and taxonomy data for the full-length sequences found in
#' Greengene's SSU (16S/18S) database version 20_10 for use with the naive
#' Bayesian classifier as implemented in the `{phylyotypr}` R package. This
#' includes 329,175 bacterial and 2,094 archaeal sequences. Originally released
#' by the biocore group in October 2022. Note that the biocore version includes
#' species names, but these have been removed from the mothur version of the
#' database. See the pages in the "Sources" section for more information.
#'
#' @format
#' A data frame with 3 columns. Each row represents a different
#' sequence:
#' \describe{
#'   \item{id}{Sequence accession identifier}
#'   \item{sequence}{DNA sequence string}
#'   \item{taxonomy}{Taxonomic string with each level separated with a `;`}
#' }
#'
#' @source
#' * [mothur-formatted files](https://mothur.org/wiki/greengenes2-formatted_databases/)
#' * [Description of how mothur-formatted files were generated](https://mothur.org/blog/2024/greengenes2_2020_10/)
#' * [Biocore's release notes for this version](https://github.com/biocore/greengenes2/)
"gg_20_10"

#' @rdname gg_20_10
"gg_20_10"


#' Greengenes training set 13_8
#'
#' The sequence and taxonomy data for the full-length sequences found in
#' Greengene's SSU (16S/18S) database version 13_8 for use with the naive
#' Bayesian classifier as implemented in the `{phylyotypr}` R package. This
#' includes 329,175 bacterial and 2,094 archaeal sequences. Originally released
#' by the biocore group in August 2013. See the pages in the "Sources" section
#' for more information.
#'
#' @format
#' A data frame with 3 columns. Each row represents a different
#' sequence:
#' \describe{
#'   \item{id}{Sequence accession identifier}
#'   \item{sequence}{DNA sequence string}
#'   \item{taxonomy}{Taxonomic string with each level separated with a `;`}
#' }
#'
#' @source
#' * [mothur-formatted files](https://mothur.org/wiki/greengenes-formatted_databases/)
#' * [Description of how mothur-formatted files were generated](https://mothur.org/blog/2024/greengenes2_2020_10/)
#' * [Biocore's release notes for this version](https://github.com/biocore/greengenes2/)
"gg_13_8"



#' Greengenes training set 13_8
#'
#' The sequence and taxonomy data for the full-length sequences found in
#' Greengene's SSU (16S/18S) database version 13_8 for use with the naive
#' Bayesian classifier as implemented in the `{phylyotypr}` R package. This
#' includes 329,175 bacterial and 2,094 archaeal sequences. Originally released
#' by the biocore group in May 2013. See the pages in the "Sources" section
#' for more information.
#'
#' @format
#' A data frame with 3 columns. Each row represents a different
#' sequence:
#' \describe{
#'   \item{id}{Sequence accession identifier}
#'   \item{sequence}{DNA sequence string}
#'   \item{taxonomy}{Taxonomic string with each level separated with a `;`}
#' }
#'
#' @source
#' * [mothur-formatted files](https://mothur.org/wiki/greengenes-formatted_databases/)
"gg_13_5"



#' Greengenes training set 13_8
#'
#' The sequence and taxonomy data for the full-length sequences found in
#' Greengene's SSU (16S/18S) database version 13_8 for use with the naive
#' Bayesian classifier as implemented in the `{phylyotypr}` R package. This
#' includes 202,421 bacterial and archaeal sequences. Originally released by the
#' Greengenes website in May 2013 (now defunct). See the pages in the "Sources"
#' section for more information.
#'
#' @format
#' A data frame with 3 columns. Each row represents a different
#' sequence:
#' \describe{
#'   \item{id}{Sequence accession identifier}
#'   \item{sequence}{DNA sequence string}
#'   \item{taxonomy}{Taxonomic string with each level separated with a `;`}
#' }
#'
#' @source
#' * [mothur-formatted files](https://mothur.org/wiki/greengenes-formatted_databases/)
"gg_13_5"


#' Greengenes training set 12_1
#'
#' The sequence and taxonomy data for the full-length sequences found in
#' Greengene's SSU (16S/18S) database version 12_1 for use with the naive
#' Bayesian classifier as implemented in the `{phylyotypr}` R package. This
#' includes 84,414 bacterial and archaeal sequences. Originally released by the
#' Greengenes website in January 2012 (now defunct) and published by Werner et al.
#' See the pages in the "Sources" section for more information.
#'
#' @format
#' A data frame with 3 columns. Each row represents a different
#' sequence:
#' \describe{
#'   \item{id}{Sequence accession identifier}
#'   \item{sequence}{DNA sequence string}
#'   \item{taxonomy}{Taxonomic string with each level separated with a `;`}
#' }
#'
#' @source
#' * [mothur-formatted files](https://mothur.org/wiki/greengenes-formatted_databases/)
#' * [Werner et al.](https://pubmed.ncbi.nlm.nih.gov/21716311/)
"gg_12_1"
